pathing/config_file_windows.go

```diff
@@ -2,7 +2,29 @@
 
 package pathing
 
+import (
+	"log"
+	"os"
+	"path/filepath"
+)
+
 const (
 	defaultConfigFile = "packer.config"
 	defaultConfigDir  = "packer.d"
 )
+
+func configDir() (path string, err error) {
+	var dir string
+	if cd := os.Getenv("PACKER_CONFIG_DIR"); cd != "" {
+		log.Printf("Detected config directory from env var: %s", cd)
+		dir = filepath.Join(cd, defaultConfigDir)
+	} else {
```

Comment:
I'd make of this one a guard close:

```suggestion
	if cd := os.Getenv("PACKER_CONFIG_DIR"); cd != "" {
		log.Printf("Detected config directory from env var: %s", cd)
		dir = filepath.Join(cd, defaultConfigDir)
		return dir, nil
	}
```